docker-compose.yml

```diff
@@ -1,5 +1,16 @@
 version: '1'
 services:
+  client:
+    image: node:18-alpine
```

Comment:
the naming here is actually the name that Docker Compose will use for the output of your build, so I'd rather call it something more personalized, like directus/client or something alike

Reply:
You can also just drop the image line, and docker compose will find a name by itself